internal/pkg/deploy/cloudformation/stack/env_integration_test.go

```diff
@@ -34,6 +34,7 @@ type: Environment
 # All these comments should be deleted.
 http:
   public:
+    restrict_alb_ingress_to_cf: false
```

Comment:
How do we do this? It will need to query for a prefix list to test against, and I don't know how to not write that explicity into the `template-with-imported-certs-observability.yml` file

Reply:
right right it is controlled by https://github.com/aws/copilot-cli/pull/3737/files#diff-031da705d16a975e87d8537638fac3cc2e4dfbc8411ff9b184cb7b28666914b9R101 instead. Can we specify `PrefixListIDs` [here](https://github.com/aws/copilot-cli/blob/45bdcc3549aa7a85ec3f08170ee44e5006ebe323/internal/pkg/deploy/cloudformation/stack/env_integration_test.go#L45)?